NEWS.md

```diff
@@ -108,6 +117,12 @@
   the first argument of an S3 method (which is necessarily evaluated
   in order to detect which class to dispatch to). (#295, #300).
 
+* Parentheses around `!!` are automatically removed. This is useful
+  for unquoting function names: `(!! sym("name"))(arg)`. If you need
+  the parenthesis wrapping, you can add a second layer since only one
```

Comment:
Why would you need it?

Reply:
I was thinking about generating code for some DSLs such as lme4. Just tried it with lme4 and it doesn't require explicit parentheses though. I'll remove the note.